Cargo.toml

```diff
@@ -167,3 +167,8 @@ panic = 'abort'
 strip = true
 #opt-level = 'z' # only have smaller size after strip
 rpath = true
+
+[patch."https://github.com/fufesou/rdev"]
+#rdev = { path = "../rdev" }
+rdev = { git = "https://github.com/sj6219/rdev", branch = "sigma" }
+
```

Comment:
Please submit a PR to fufesou/rdev.

Reply:
Could you please solve this?

Reply:
I submitted a PR, but my PR has not yet been accepted.

Reply:
Sorry for late, the PR is accepted now.

Reply:
Thanks, let me notify him.